test/cpp/jit/test_gpu.cpp

```diff
@@ -2971,88 +2973,86 @@ void testGPU_FusionSimpleBCast() {
 }
 
 void testGPU_FusionSimpleGemm() {
-  {
```

Comment:
note the extra { ... } around the body of the function, which resulted in a indent update